### PR TITLE
Article: Bootstrap

### PR DESCRIPTION
**Pull request by theflametrooper**
Refer #707

### CODE REVIEW

File: Bootstrap.md [hunk 1]
```diff
@@ -1,5 +1,30 @@
...
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Bootstrap Template</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
+  </head>
+  <body>
+    <h1>Hello World</h1>
```
Comment (alayek) [inline Bootstrap.md]: Maybe add a comment here that the developer should edit the body of HTML here

**Comment by alayek** [comment]
# LGTM :+1: 

@abhisekp @Rafase282 might wanna take a look.

File: Bootstrap.md [hunk 2]
```diff
@@ -1,5 +1,30 @@
-A frontend [CSS](https://github.com/bothelpers/kbase/wiki/css) framework.
+# Bootstrap
 
-You can find out more [here](http://getbootstrap.com/)
+### Getting Started
 
-> related: css, frameworks
+Here is a simple HTML template which includes the latest compiled and minified CSS and Javascript for the Bootstrap library.
+We have used a CDN in this example, but you can checkout other ways of installing Bootstrap [here](http://getbootstrap.com/getting-started/#download).
+
```
Comment (abhisekp) [inline Bootstrap.md]: Add a header 2 here
```md
## (put a space after ##)
```

File: Bootstrap.md [hunk 3]
```diff
@@ -1,5 +1,30 @@
-A frontend [CSS](https://github.com/bothelpers/kbase/wiki/css) framework.
+# Bootstrap
 
-You can find out more [here](http://getbootstrap.com/)
+### Getting Started
```
Comment (abhisekp) [inline Bootstrap.md]: Add a Bootstrap intro and then add a blank header 2 after it

<mini intro to Bootstrap>

```md
## (add a space after ##)
```

**Comment by abhisekp** [comment]
@theflametrooper is this WIP? :construction: :construction_worker: 

Shouldn't there be a very basic tutorial for Bootstrap?

----
This is a nice tutorial to add http://www.tutorialrepublic.com/twitter-bootstrap-tutorial/

**Comment by theflametrooper** [comment]
@abhisekp It's finished. It's just a simple guide and not going to include any form of tutorial. I will add your link :)

**Comment by theflametrooper** [comment]
@alayek Should be good to go! Just be sure to squash and then :shipit:

**Comment by Rafase282** [comment]
# LGTM

**Pull request merged**